roles/kubernetes-apps/ansible/templates/nodelocaldns-config.yml.j2

```diff
@@ -75,6 +75,9 @@ data:
         prometheus {% if nodelocaldns_bind_metrics_host_ip %}{$MY_HOST_IP}{% endif %}:{{ nodelocaldns_prometheus_port }}
     }
     .:53 {
+{% if nodelocaldns_additional_configs is defined %}
```

Comment:
Should `in-addr.arpa:53 ` and ` ip6.arpa:53` require these configs if we need these configs?

Reply:
It depends on the intended use of this variable I suppose. I kept this consistent with `coredns_additional_configs`:

https://github.com/kubernetes-sigs/kubespray/blob/d23753e9f7ff10de50d3ddb81354f22c484cd90d/roles/kubernetes-apps/ansible/templates/coredns-config.yml.j2#L37-L40